modules/preferences/main/index.coffee

```diff
@@ -207,7 +207,10 @@ class Preferences extends hx.EventEmitter
       modal: modal
     }
 
-    @locale moment?.locale() or navigator.language or 'en'
+    defaultLocaleId = moment?.locale() or navigator.language or 'en'
```

Comment:
The line below where it checks 'localeLookup' could just check `defaultLocale` first to save _always_ looking up the locale when we know 'en' is in the list